encoding/mkv.md

```diff
@@ -0,0 +1,505 @@
+Same goes for the ED.
+
+```py
+import vapoursynth as vs
+core = vs.core
+
+bdmv01 = core.lsmas.LWLibavSource('ep01.m2ts')
+bdmv02 = core.lsmas.LWLibavSource('ep02.m2ts')
+...
+bdmv13 = core.lsmas.LWLibavSource('ep13.m2ts')
+bdmvNCED = core.lsmas.LWLibavSource('nced.m2ts')
+bdmvNCOP = core.lsmas.LWLibavSource('ncop.m2ts')
+
+ep01 = bdmv01[:30042]+bdmv01[32201:34046]
+op = bdmv01[30042:32201]  # the first time the OP plays is during episode 01
+ep02 = bdmv02[:1678]+bdmv02[3837:30978]+bdmv02[33111:34023]
+ed = bdmv02[30978:33111]  # the first time the ED plays is during episode 02
+ep03 = bdmv03[:1942]+bdmv03[4101:30474]+bdmv03[32609:34023]
+ep04 = bdmv04[:2566]+bdmv04[4723:30976]+bdmv04[33109:34021]
+ep05 = bdmv05[:840]+bdmv05[2997:31266]+bdmv05[33399:34023]
+ep06 = bdmv06[:1702]+bdmv06[3861:30257]+bdmv06[32392:34022]
+ep07 = bdmv07[:1103]+bdmv07[3261:31218]+bdmv07[33351:34023]
+ep08 = bdmv08[:3923]+bdmv08[6090:30449]+bdmv08[32584:34022]
+ep09 = bdmv09[:720]+bdmv09[2878:31001]+bdmv09[33134:34022]
+ep10 = bdmv10[:792]+bdmv10[2949:30161]+bdmv10[32296:34022]
+ep11 = bdmv11[:288]+bdmv11[2446:29082]+bdmv11[31216:34021]
+ep12 = bdmv12[:34213]  # episode 12 has no OP/ED
+ep13 = bdmv13[2158:30474]
+nced = bdmvNCED[:2133]
+ncop = bdmvNCOP[:2158]
+```
+
+*Note: the second number of the Python trims is NOT inclusive.*
+
+
+## Trimming the audio + generating chapters
+
+**Note:** *the frame numbers used in acsuite's `octrim`
+and `eztrim`
+are from the __untrimmed source clips__.
+The exact syntax is explained in the docstrings for `acsuite.py`.*
+
+
+### Non-chapter files (OP/ED)
+
+For the OP/ED, NCOP/NCED we will use the frame ranges we found above
+in the [VapourSynth script](#starting-with-trimming-the-video).
+Since there are no chapters for these four clips, we will be using acsuite's
+`eztrim`.
+
+In order to save time, we will be using `acsuite.audio_trim`
+as a wrapper for `eztrim` and FFmpeg's audio extraction.
+
+*If the desired audio track is not the first,
+add `track_no=#` to the `audio_trim` arguments.*
+
+```py
+import acsuite as ac
+
+# op = bdmv01[30042:32201] --> eztrim takes these frames directly (30042, 32201)
+# ed = bdmv02[30978:33111] --> (30978, 33111)
+# nced = bdmvNCED[:2133] --> (0, 2133)
+# ncop = bdmvNCOP[:2158] --> (0, 2158)
+
+ac.audio_trim('ep01.m2ts', (30042, 32201), ez=True, name='op')
+ac.audio_trim('ep02.m2ts', (30978, 33111), ez=True, name='ed')
+ac.audio_trim('nced.m2ts', (0, 2133), ez=True)
+ac.audio_trim('ncop.m2ts', (0, 2158), ez=True)
+```
+
+The above lines can be ran individually in a new Python instance
+in your working directory.
+After this is complete,
+there will be 8 new files in your directory:
+
+- `ed_cut.wav` *
+- `ep01.wav` (delete)
+- `ep02.wav` (delete)
+- `nced_cut.wav` *
+- `nced.wav` (delete)
+- `ncop_cut.wav` *
+- `ncop.wav` (delete)
+- `op_cut.wav` *
+
+Files marked with an asterisk * are our trimmed audio files.
+The rest are artifacts from the FFmpeg extraction that can be deleted now.
```

Comment:
Is the script aware of these intermediate files for caching purposes? If it is not, you might as well delete them from the script.

Reply:
Yea, it is but how should I remove them cross-platform?

Reply:
I don't see what part of this argument is cross-platform? `os.unlink` works on all. Regardless, if these files can be considered a cache, you can leave them be or cache them in a subdirectory to keep things more organized. From a workflow standpoint, however, I hate having to delete intermediate files I'm not going to look at again and rather have the tool that created them do that for me.